rotkehlchen/chain/gnosis/node_inquirer.py

```diff
@@ -53,7 +52,7 @@ def __init__(
             contracts=contracts,
             rpc_timeout=rpc_timeout,
             contract_multicall=contracts.contract(string_to_evm_address('0xcA11bde05977b3631167028862bE2a173976CA11')),
-            contract_scan=BALANCE_SCANNER,
+            contract_scan=contracts.contract(string_to_evm_address('0x54eCF3f6f61F63fdFE7c27Ee8A86e54899600C92')),
```

Comment:
you are using this across many places. Just define it as a constant in a common place.